packages/grid/src/GridMetricCalculator.ts

```diff
@@ -803,7 +819,7 @@ export class GridMetricCalculator {
       lastLeft = right;
     }
     let x = 0;
-    while (lastLeft > 0) {
+    while (lastLeft >= 0) {
```

Comment:
This is reverting your fix from https://github.com/deephaven/web-client-ui/pull/940, and that problem is now back. Need some way to handle that case.